src/runtime/startup/c/wrapper.c

```diff
@@ -841,7 +862,6 @@ __attribute__((constructor)) static void init(void) {
                 }
             }
         }
-#ifdef NS_CLONE_NEWCGROUP
```

Comment:
Similar question here, why remove check for if newcgroup supported? I assume has something to do with addition of `#ifndef` blocks at top of file?

Reply:
The logic here is to fail rather than don't execute the code because not supported. More explicit for debug or user issues